samples/tab-ui-templates/ts/.gitignore

```diff
@@ -84,3 +84,25 @@ yarn-error.log*
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+######
+
+# TeamsFx files
```

Comment:
Please refer to https://github.com/formulahendry/Microsoft-Teams-Samples/blob/junhan/v3/samples/tab-ui-templates/ts/.gitignore

Reply:
changed